index.bs

```diff
@@ -391,6 +391,46 @@ Implementers of this API are expected to be familiar with the <a href="https://g
 
 
 # Programming Model # {#programming-model}
+## Overview ## {#programming-model-overview}
+
+At the heart of neural networks is a computational graph of mathematical operations.
+These operations are the building blocks of modern machine learning technologies in
+computer vision, natural language processing, and robotics.
+The WebNN API is a specification for constructing, compiling, and executing computational
+graphs of neural networks.
+
+The {{MLGraph}} interface represents a compiled computational graph (that is, a model) and exposes
+a compute method to perform inference.
+
+The {{MLGraphBuilder}} interface serves as a builder (factory) to create a {{MLGraph}}.
+The {{MLOperand}} interface is a static (compile-time) representation of runtime values,
+which include input-values for inference, constants (including trained weights)
+used for inference, intermediate values (often referred to as activations) computed
+during inference, as well as the output values of inference.
```

Comment:
Thanks @gramalingam. The notion of `MLOperand` is a bit harder to explain. But I think it helps to draw a contrast between what is an operand and what is a tensor. In my mind, an operand is really a *description* of the data and not the data itself i.e. we need to bind the data to operands before executing the model, etc. I sense that's probably what you try to say in your wordings around "is a static (compile-time) representation"?

Reply:
Thanks, addressed. Apologies for the delay in responding, something is wrong with my notification/email, I didn't see the notifications for these comments.